Tools/CMake/AMReX_Options.cmake

```
@@ -206,16 +206,19 @@ endif ()
 option( ENABLE_PIC "Build position-independent code" OFF)
 print_option( ENABLE_PIC )
 
-option(ENABLE_FPE "Enable Floating Point Exceptions checks" OFF)
-print_option( ENABLE_FPE )
-
+set(ENABLE_FPE_DEFAULT OFF)
+set(ENABLE_ASSERTIONS_DEFAULT OFF)
 if ( "${CMAKE_BUILD_TYPE}" MATCHES "Debug" )
```

```suggestion
option(ENABLE_FPE "Enable Floating Point Exceptions checks" OFF)
print_option( ENABLE_FPE )
```

I would leave FPE options as an opt out no matter what the build type is.

---

Yeah, I don't know - maybe something you and @WeiqunZhang can weight the pro's and cons in your developer meetings, I have no real opinion on it.

I am okay to keep it enabled automatically in Debug, as done in GNUmake:
https://github.com/AMReX-Codes/amrex/blob/7e5c619d5ce21907e6f4dbcff17fef3c3a10d992/Tools/GNUMake/comps/gnu.mak#L77-L80